.github/workflows/benchmarks.yml

```diff
@@ -46,18 +46,18 @@ jobs:
 
       - name: "Install lowest dependencies"
         if: ${{ matrix.dependencies == 'lowest' }}
-        run: "composer update --prefer-lowest --no-interaction --no-progress --no-suggest"
+        run: "composer update --prefer-lowest --root-reqs --no-interaction --no-progress"
```

Comment:
We should really not use `--root-reqs` when testing upgrades/downgrades: what users get is a full "lowest" or "newest", not just dependencies we declared.

For our own stability needs, there is the `locked` pipeline instead.

Reply:
I modified it because the tests are failing because of this: https://github.com/Roave/BetterReflection/runs/3600471383?check_suite_focus=true

Is there a better way how to fix it?

Reply:
Possibly declaring a conflict with earlier `thecodingmachine/safe`, since it seems like it will break in prod anyway, when in use

Reply:
Clever 👍

Reply:
https://github.com/Roave/BetterReflection/pull/750/commits/0a86763a8ce0b2cb53d199df775b3d07ebe49d3a